test/e2e-tests-standing-order.sh

```diff
@@ -17,8 +17,36 @@ step "Place Sell Order" \
 step "Place standing order in current Auction" \
 "npx truffle exec scripts/standing_order.js 0 1 2 1 1"
 
-step_with_retry "Check standing order has been recorded" \
-"mongo dfusion2 --eval \"db.standing_orders.findOne({accountId:0, batchIndex:0, orders: [ { buyToken:1, sellToken:2, buyAmount:'1000000000000000000', sellAmount:'1000000000000000000' }]})\" | grep ObjectId"
+step_with_retry "Check mongo standing order has been recorded" \
+"mongo dfusion2 --eval \"db.standing_orders.findOne({accountId:0, batchIndex:0, validFromAuctionId:0, orders: [ { buyToken:1, sellToken:2, buyAmount:'1000000000000000000', sellAmount:'1000000000000000000' }]})\" | grep ObjectId"
+
+step_with_retry "Check graph standing order batch has been recorded" \
+"source ../test/utils.sh && query_graphql \
```

Comment:
Wouldn't this check be enough, since you are already listing the order here? Technically you could grep for multiple fields at the same time.

Might make the tests a little bit more concise if we don't have to check two queries per standing order.

Reply:
You cannot filter by subentities right now. So the second query verifies the values of the order, I don't see how I can do that easily with grep.

Also, I think it's good to check the query for both entities

Reply:
This should do the same as the two queries in one. We basically ask for all orders on the standing order, select all relevant fields and then match the expected json output.

```sh
step_with_retry "Check graph standing order batch has been updated" \
"source ../test/utils.sh && query_graphql \
    \"query { \
        standingSellOrderBatches(where: { \
          accountId: 0, \
          batchIndex: 1, \
          validFromAuctionId: 2 \
        }) { \
        orders { \
            buyToken \
            sellToken \
            buyAmount \
            sellAmount \
        } \
      } \
    }\" | grep \"buyAmount.:.1000000000000000000.,.buyToken.:2,.sellAmount.:.2000000000000000000.,.sellToken.:1\""
```

Reply:
You are a grep virtuoso. I'll make the change